DashBoard_PowerBox_V2_Arduino/DashBoard_PowerBox_V2/DashBoard_PowerBox_V2.ino

```diff
@@ -43,6 +43,24 @@ float PWR_TOTAL=0.00;
 float NTC_DELAY;
 double ACS_resolution;
 
+#define QUEUELENGTH         5             // number of commands that can be saved in the serial queue
+#define MAXCOMMAND          21            // max length of a command
+#define EOFSTR              '\n'
+#define EOCOMMAND           '#'           // defines the end character of a command
+#define SOCOMMAND           '>'           // defines the start character of a command
```

Comment:
not needed but a suggestion for something that is currently missing, if implemented there should be a change in serialEvent() too